app/_hub/reedelk/reedelk-transformer/index.md

```diff
@@ -0,0 +1,205 @@
+
+To use the Reedelk plugin for Kong, you must first install the Reedelk IntelliJ
+IDEA flow designer plugin using either of the following options:
+
+- **IntelliJ Marketplace**: From **IntelliJ Preferences > Plugin > Marketplace**,
+  search for `Reedelk`, install the plugin, and restart IntelliJ.
+
+- **Manual install**: From **IntelliJ Preferences > Plugin > Settings icon >
+  Install Plugin From Disk**, and restart IntelliJ.
+
+### Install Kong Plugin Steps
+
+1. Clone the `kong-plugin-reedelk-transformer`:
+
+   ```bash
+      git clone https://github.com/reedelk/kong-plugin-reedelk-transformer.git
+      cd kong-plugin-reedelk-transformer
+   ```
+
+2. Install the module `kong-plugin-reedelk-transformer`:
+
+   ```bash
+   luarocks install kong-plugin-reedelk-transformer-0.1.0-1.all.rock
+   ```
+
+3. Add the custom plugin to `kong.conf`:
+
+   ```
+   plugins = ...,reedelk-transformer
+   ```
+
+4. [Restart](/2.1.x/cli/#kong-restart) Kong.
+
+### Building and Packaging the Kong Plugin
+
+1. The [LuaRocks](http://luarocks.org) package manager must be
+   [installed](https://github.com/luarocks/luarocks/wiki/Download).
+
+2. Build the plugin locally (based on the `.rockspec` in the current directory).
+
+   ```bash
+   $ luarocks make
+   ```
+
+3. Package the plugin.
+
+   ```bash
+   $ luarocks pack kong-plugin-reedelk-transformer 0.1.0-1
+   ```
+
+### Hello World Example
+
+This example uses a pre-packaged `kong-reedelk` docker image with Kong and
+`reedelk-transformer` plugin already installed. You can find the `kong-reedelk`
+docker images on [Dockerhub](https://hub.docker.com/repository/docker/reedelk/kong-reedelk-transformer-plugin).
+
+The `kong-reedelk` image is pre-configured to use the following `kong.yml` file,
+which defines an upstream service mapped on Route `http://localhost:8000/transform`
+and invokes a downstream transformer with the `reedelk-transformer` plugin. The
+configured downstream transformer integration flow URL is:
+
+```
+http://host.docker.internal:8282/api/message
+```
+
+#### Prerequisites
+
+- [Docker](https://www.docker.com/) must be [installed](https://www.docker.com/get-started).
+- [IntelliJ IDEA IDE](https://www.jetbrains.com/idea/) must be
+  [installed](https://www.jetbrains.com/idea/download).
+- [Reedelk IntelliJ Flow Designer Plugin](https://www.reedelk.com/documentation/intellijplugin)
+  must be [installed](https://www.reedelk.com/documentation/intellijplugin) on
+  your IntelliJ distribution.
+
+#### Run the example
+
+In the following steps, you run the `kong-reedelk` docker image, and then create a
+new Reedelk project that contains the integration flow invoked by the
+downstream transformer.
+
+1. Run the `kong-reedelk` Docker image:
+
+   ```
+   docker run -d --name kong-reedelk-transformer-plugin \
+            -p 8000:8000 \
+            -p 8443:8443 \
+            -p 127.0.0.1:8001:8001 \
+            -p 127.0.0.1:8444:8444 \
+            reedelk/kong-reedelk-transformer-plugin:1.0.0
```

Comment:
@tlblessing Hi Terry, I would like to change this line so that the version of the Docker image from Dockerhub is not hardcoded anymore but it always takes the latest one. Could you please replace with the suggestion below? Thank you
```suggestion
            reedelk/kong-reedelk-transformer-plugin:latest
```

Reply:
Done in commit b6e39ef